cloud_storage_handler/main.py

```diff
@@ -39,13 +29,50 @@ def init_app() -> FlaskApp:
 
     foca = Foca(
         config_file=config_path,
+        custom_config_model="cloud_storage_handler.custom_config.CustomConfig",
     )
-    return foca.create_app()
+    foca_config = foca.conf
+    minio_config = foca_config.custom.minio
+    logger.info(minio_config)
+
+    # Create the Connexion FlaskApp instance
+    return foca.create_app(), minio_config
 
 
 def main() -> None:
     """Run FOCA application."""
-    app = init_app()
+    app, minio_config = init_app()
+    foca_app = app.app
+
+    if minio_config is None:
+        raise KeyError("Custom configuration for MinIO not found in Flask app config.")
```

Comment:
With the above changes this should not be needed ig as is it was to break it would break above itself.